src/lib.rs

```diff
@@ -638,7 +638,7 @@ pub fn build_with_subcommand(parameters: CompileParameters) -> Result<(), Diagno
             copy_libs_to_build(&project.libraries)?;
 
             let compile_options = CompileOptions {
-                output: project.output,
+                output: project.output.unwrap_or_else(|| String::from("proj.so")),
```

Comment:
We have a mechanism to get the correct name here, we use it in build with params